tools/build_boot_image

```diff
@@ -45,7 +45,7 @@ bi_make_iso=0
 bi_proforma_only=0
 bi_nocleanup=0
 bi_imgsz_gb="2"
-bi_imgsz=$(( $bi_imgsz_gb * 1000000000 ))
+bi_imgsz="1940000256"
```

Comment:
Please add a comment as to why we have such a weird value.

Reply:
I committed a comment just above the initialization of variables section to explain the value. It seemed to be in keeping with how other documentation was organized. Let me know if you would prefer it on the line directly above or following on the same line instead.